src/main/java/pl/anasz/cinema/repositories/SeatsRepository.java

```diff
@@ -0,0 +1,23 @@
+package pl.anasz.cinema.repositories;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+import org.springframework.stereotype.Repository;
+import pl.anasz.cinema.enities.Seat;
+
+import java.util.List;
+import java.util.Optional;
+
+@Repository
+public interface SeatsRepository extends JpaRepository<Seat,Long> {
+
+    @Query("SELECT s.id FROM Seat s WHERE s.room.id = :roomId")
```

Comment:
Tych querasów można nie pisać, jeśli są zgodne z JPA.

Reply:
Tak, zgodzę się, że nie trzeba, jednak chciałam mieć tutaj jednorodność ;).

Reply:
W porządku, jest to dla mnie zrozumiałe.